components/feature/app-links/src/main/java/mozilla/components/feature/app/links/AppLinksFeature.kt

```diff
@@ -44,7 +45,8 @@ class AppLinksFeature(
     private val launchInApp: () -> Boolean = { false },
     private val useCases: AppLinksUseCases = AppLinksUseCases(context, launchInApp),
     private val failedToLaunchAction: () -> Unit = {},
-    private val loadUrlUseCase: SessionUseCases.DefaultLoadUrlUseCase? = null
+    private val loadUrlUseCase: SessionUseCases.DefaultLoadUrlUseCase? = null,
+    private val loadUrlWhenFailed: Boolean = false
```

Comment:
Is the `loadUrlWhenFailed` boolean needed? It looks like this can be controlled by `loadUrlUseCase` being `null` or not? 🤔

Reply:
Hmm, I see what you mean.  This doesn't really need a choice, as long as we can load the URL we should load it when it fails.  I'll update.  Thanks,